pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go

```diff
@@ -481,8 +481,8 @@ func (o *JobRunTestCaseAnalyzerOptions) Run(ctx context.Context) error {
 		return fmt.Errorf("error creating output directory %q: %w", outputDir, err)
 	}
 
-	// if it hasn't been more than hour since the jobRuns started, the list isn't complete.
-	readyAt := o.jobRunStartEstimate.Add(1 * time.Hour)
+	// if it hasn't been more than two hours since the jobRuns started, the list isn't complete.
+	readyAt := o.jobRunStartEstimate.Add(2 * time.Hour)
```

Comment:
I think there is one more to hit unless you just want to change testcaseanalyzer

https://github.com/openshift/ci-tools/blob/2b99a76b22b0bc656632df0c7d8363afc22afde6/pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer.go#L73